discord_bot/bot.py

```diff
@@ -0,0 +1,70 @@
+import os
+from dotenv import load_dotenv
+import asyncio
+import interactions
+
+# Real VAIT server id to be used and
+# THIS SHOULD BE ASSIGNED to `scopes` WHEN DEPLOYED
+vait_guild_id = [999999999]
```

Comment:
Can this be abstracted into an envvar,eg: DISCORD_GUILD_ID for example

Reply:
The guild_id is also not used. If it's not used please remove.

Reply:
Done, move to .env.example